lib/openstax/aws/stack.rb

```diff
@@ -292,6 +292,10 @@ def resource(logical_id)
         name = stack_resource.physical_resource_id
         client = Aws::AutoScaling::Client.new(region: region)
         Aws::AutoScaling::AutoScalingGroup.new(name: name, client: client)
+      when "Aws::RDS::DBInstance"
+        db_instance_identifier = stack_resource.physical_resource_id
+        client = Aws::RDS::Client.new(region: region)
+        Aws::RDS::DBInstance.new(db_instance_identifier: db_instance_identifier, client: client)
```

Comment:
Note that the ASG code right above is returning an instance of a class direct from the AWS Ruby SDK -- `Aws::AutoScaling::AutoScalingGroup` is a class provided by AWS; it isn't our wrapper class.  It could be, but for whatever reason we didn't return an instance of our wrapper class there.

On this code we've added, we probably do want to return an instance of our wrapper class.  So this line would instead be

```ruby
OpenStax::Aws::RdsInstance.new(db_instance_identifier: db_instance_identifier, region: region)
```

and line 297 can go away (it is already inside our wrapper class on line 11).

As a Ruby thing, you could also say `RdsInstance.new...` instead of `OpenStax::Aws::RdsInstance.new...` because this code is inside the `module OpenStax::Aws` declaration on line 1 of this file.